src/models/tools/shared-model-document-manager.ts

```diff
@@ -9,7 +9,7 @@ export interface ISharedModelDocumentManager extends ISharedModelManager {
   setDocument(document: DocumentContentModelType): void;
 }
 
-export const SharedModelDocumentManager = 
```

Comment:
This has changed to a MobX class instead of a MST model, so master will need to be merged into this branch and the conflicts fixed.

Reply:
Let's talk about it before you do that. It might be better to review/merge the geometry/table PR first and then do the merge from master and address conflicts.